lib/OpenQA/WebSockets/Server.pm

```diff
@@ -216,6 +216,15 @@ sub _message {
         return;
     }
 
+    # This is to make sure that no worker can skip the _registration.
+    if (($worker->{db}->get_websocket_api_version() // 0) != INTERFACE_VERSION) {
+        log_debug("Received a message from an incompatible worker");
```

Comment:
This is a situation that should not happen too often, but should be seen on the logs... anything above debug should do.

Reply:
done